changelogs/fragments/ansible-test-added-macos-15.3.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+minor_changes:
+- ansible-test - Added a macOS 15.3 remote VM.
```

Comment:
Don't forget to mention this is a replacement for macOS 14.3.

Reply:
Oh, I copied the old note. Will do.

Reply:
Done. Marking as resolved.